### PR TITLE
New airdrop mechanism

### PR DESCRIPTION
**Pull request by larry0x**
- Delete the airdrop contract
- Instead, assign users the coin balances directly in the genesis state. Users no longer need to manually claim. Ideally, they can simply open Terra Station wallet, switch the chain to Mars, and the MARS token balance should appear in their wallet.

### CODE REVIEW

File: scripts/add_wasm_msgs.ts [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-// This should be a Cosmos SDK native multisig created off-chain
-const DEPLOYER = "mars15mwq8jc7sf0r8hu6phahfsmqg3fagt7ysyd3un";
-
-// Contract addresses are derived determinictically from code id and instance id, which we can
-// predict beforehand.
-const CONTRACTS = {
-  VESTING: "mars14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9smxjtde",
-  AIRDROP: "mars1nc5tatafv6eyq7llkr2gv50ff9e22mnf70qgjlv737ktmt4eswrqhnhf0l",
-  DELEGATOR: "mars17p9rzwnnfxcjp32un9ug7yhhzgtkhvl9jfksztgw5uh69wac2pgs0gfvxm",
-  VESTING_OWNER: "mars1ghd753shjuwexxywmgs4xz7x2q732vcnkm6h2pyv9s6ah3hylvrqn7y4x6",
```
Comment (joshlopes) [inline scripts/add_wasm_msgs.ts:19]: Not sure if my other comment worked or if it got lost -- apologies to repeat the same remark

@larry0x have you noticed that DELEGATOR contract address was removed and it's address was used for the VESTING_OWNER address which is now the DELEGATOR address? Just to confirm that was intended.

Comment (larry0x) [reply scripts/add_wasm_msgs.ts:19]: Yes, this is intended.

To be more specific, `AIRDROP` contract was removed, `DELEGATOR` takes `AIRDROP`'s address, and `VESTING_OWNER` takes `DELEGATOR`'s address.

This has to do with [how CosmWasm derives contract addresses](https://github.com/CosmWasm/wasmd/blob/a347ace2ff41539fe06c68168bc6f28d6ca9fa52/x/wasm/keeper/addresses.go#L35).

Basically, the wasm module identifies each contract by two numbers, the "code_id" and the "instance_id":
- The first code ever to be stored on-chain gets "code_id" of `1`, the second gets `2`, and so on
- The first contract ever to be instantiated gets |instance_id" of `1`, the second gets `2`, and so on

The contract address is then calculated by

```plain
contract_address := hash("wasm" + code_id + instance_id)
```

Previous to this PR, the contracts have the following ids:

| contract | code id | instance id |
| --- | --- | --- |
| vesting | `1` | `1` |
| airdrop | `2` | `2` |
| delegator | `3` | `3` |
| vesting_owner | `4` | `4` | 

This PR deletes the airdrop contract, so the ids change to:

| contract | code id | instance id |
| --- | --- | --- |
| vesting | `1` | `1` |
| delegator | `2` | `2` |
| vesting_owner | `3` | `3` | 

Therefore, `DELEGATOR` takes what's previously `AIRDROP`'s address, and `VESTING_OWNER` takes what's previously `DELEGATOR`'s address.

Comment (larry0x) [reply scripts/add_wasm_msgs.ts:19]: If you download the `genesis.json` to your `~/.mars/config`, you can list the contracts with the following command to verify the addresses are correct:

```bash
$ marsd genesis add-wasm-message list-contracts
[
 {
  "contract_address": "mars14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9smxjtde",
  "info": { ... }
 },
 {
  "contract_address": "mars1nc5tatafv6eyq7llkr2gv50ff9e22mnf70qgjlv737ktmt4eswrqhnhf0l",
  "info": { ... }
 },
 {
  "contract_address": "mars17p9rzwnnfxcjp32un9ug7yhhzgtkhvl9jfksztgw5uh69wac2pgs0gfvxm",
  "info": { ... }
 },
 {
  "contract_address": "mars1unyuj8qnmygvzuex3dwmg9yzt9alhvyeat0uu0jedg2wj33efl5q56gnrn",
  "info": { ... }
 }
]
```

Comment (joshlopes) [reply scripts/add_wasm_msgs.ts:19]: Makes sense, thanks for checking it and explaining it so well.

**Pull request merged**